### PR TITLE
Add optional image shadow props

### PR DESCRIPTION
**Pull request by seekuehe**
**Description**
This PR takes the shape shadow implementation and adapts it to images.
Related to [Issue #986](https://github.com/gitbrent/PptxGenJS/issues/986)

**Usage**
```js
let pptx = new PptxGenJS();
let slide = pptx.addSlide();

slide.addImage({
  x: 1,
  y: 1,
  path: "https://images.unsplash.com/photo-1506744038136-46273834b3fb?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMTIyMnwwfDF8c2VhcmNofDJ8fGxhbmRzY2FwZXxlbnwwfHx8fDE2NTg5MzE0MDM&ixlib=rb-1.2.1&q=80&w=1080",
  shadow: {
    type: "outer",
    angle: 90,
    blur: 10,
    color: "000000",
    offset: 4,
    opacity: 0.5
  }
});

pptx.writeFile({ fileName: "PptxGenJS-Sandbox.pptx" });

```
Sample PPT file: [PptxGenJS-Sandbox.pptx](https://github.com/pitch-io/PptxGenJS/files/9300896/PptxGenJS-Sandbox.pptx)

### CODE REVIEW

File: demos/modules/demo_image.mjs [hunk 1]
```diff
@@ -279,3 +280,19 @@ function genSlide04(pptx) {
...
 }
+
+/**
+ * SLIDE 5: Image Shadow
+ * @param {PptxGenJS} pptx
+ */
+ function genSlide05(pptx) {
+	let slide = pptx.addSlide({ sectionTitle: "Images" });
+
+	slide.addTable([[{ text: "Image with Shadow Props", options: BASE_TEXT_OPTS_L }, BASE_TEXT_OPTS_R]], BASE_TABLE_OPTS);
```
Comment (tjinauyeung) [inline demos/modules/demo_image.mjs]: Following the convention of the deck 
```suggestion
	slide.addTable([[{ text: "Image Examples: Shadows", options: BASE_TEXT_OPTS_L }, BASE_TEXT_OPTS_R]], BASE_TABLE_OPTS);
```

File: demos/modules/demo_image.mjs [hunk 2]
```diff
@@ -279,3 +280,19 @@ function genSlide04(pptx) {
...
+	slide.addNotes("API Docs: https://gitbrent.github.io/PptxGenJS/docs/api-images.html");
+	slide.slideNumber = { x: "50%", y: "95%", w: 1, h: 1, color: COLOR_BLUE };
+
+	// EXAMPLES
+	slide.addText("shadow: {type: 'outer', opacity: 0.5, blur: 20, color: '000000', offset: 20, angle: 270}", { x: 0.5, y: 0.6, w: 6.0, h: 0.3, color: COLOR_BLUE });
```
Comment (tjinauyeung) [inline demos/modules/demo_image.mjs]: Screenshot 2022-08-10 at 16 58 44
 
He uses a convention here i.e. Shadow: \`type:outer, opacity:0.5`

Comment (tjinauyeung) [reply demos/modules/demo_image.mjs]: It may be useful to have an example for each shadow type

**Comment by tjinauyeung** [comment]
Screenshot 2022-08-10 at 17 01 46

This view is still missing a description of the newly added slide.

**Review by tjinauyeung — APPROVED** [review]

**Comment by seekuehe** [comment]
PR to upstream repo: https://github.com/gitbrent/PptxGenJS/pull/1147

**Pull request closed without merging**